### ISSUE TITLE
Input field order

### ISSUE THREAD

**Issue by DamirSvrtan** [stormpath/stormpath-framework-tck#259]
Hey Nate,

I'm working on the Rails Framework integration. Rails adds two additional input fields on every form:
- `uf8` - parameter added to forms in order to force Internet Explorer (5, 6, 7 and 8) to encode its parameters as unicode.
- `authenticity_token` - CSRF protection.

Since these two fields are on top of every form, my tests are failing on the `loginFormShouldBeOrderedCorrectly` spec:

``` java
java.lang.AssertionError: expected [login] but found [authenticity_token]
```

**Comment by omgitstom** [comment]
Looks like we need to ignore hidden fields.

**Comment by edjiang** [comment]
On it.

**Comment by edjiang** [comment]
A fix is ready under PR #275! Let me know if it works for you =] @DamirSvrtan
